test/javascript/index.html

```diff
@@ -6,16 +6,16 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script src="../../node_modules/jquery/dist/jquery.js"></script>
     <script src="../../_build/assets/js/foundation.js"></script>
-    <script src="lib/mocha.js"></script>
-    <link rel="stylesheet" href="lib/mocha.css" />
+    <script src="../../node_modules/mocha/mocha.js"></script>
+    <link rel="stylesheet" href="../../node_modules/mocha/mocha.css" />
     <link rel="stylesheet" href="../../_build/assets/css/foundation-float.css">
     <link rel="stylesheet" href="../../node_modules/motion-ui/dist/motion-ui.css">
   </head>
   <body>
     <div id="mocha"></div>
-    <script src="lib/chai.js"></script>
-    <script src="lib/chai-jquery.js"></script>
-    <script src="lib/sinon.js"></script>
+    <script src="../../node_modules/chai/chai.js"></script>
```

Comment:
We use `chai` directly, so please install `chai` in devDependencies.

Reply:
That's what I did (see the package.json)?

Reply:
hmmm I did not seen that, sorry.